bin/alluxio-common.sh

```diff
@@ -10,6 +10,8 @@
 # See the NOTICE file distributed with this work for information regarding copyright ownership.
 #
 
+PORT=22
```

Comment:
Add a global var to define the remote ssh port in common.sh. In this way, we just only to change the port once.